entries/b/bowens.com.au.json

```diff
@@ -0,0 +1,16 @@
+{
+    "Bowens":{
+      "domain":"bowens.com.au",
+      "recovery": "https://www.bowens.com.au/my-account/login?redirect=%2F",
```

Comment:
The recovery element is intended to provide users documentation on how accounts can be recovered. Not an actual link to the recovery page.
```suggestion
```